var/spack/repos/builtin/packages/intel-oneapi-ipp/package.py

```diff
@@ -30,6 +30,12 @@ class IntelOneapiIpp(IntelOneApiLibraryPackage):
     )
 
     if platform.system() == "Linux":
+        version(
+            "2021.7",
```

Comment:
package number is 2021.6.1 in download url?

Reply:
The package number is correct. I do believe we have a problem with the version.